app/src/main/java/com/sun_asterisk/moviedb_49/data/model/Movie.kt

```diff
@@ -0,0 +1,47 @@
+package com.sun_asterisk.moviedb_49.data.model
+
+data class Movie(val build: MovieBuild) {
+
+    private var mId: Int? = null
```

Comment:
```suggestion
    private var id: Int? = null
```
tương tự các field ở dưới.